how-to-guides/re-run-failed-training/notebooks/re_run_failed_training.ipynb

```diff
@@ -378,7 +378,6 @@
    },
    "outputs": [],
    "source": [
-    "failed_run.stop()\n",
```

Comment:
Why remove this?

Reply:
@SiddhantSadangi the stop() for failed_run shouldn't be removed

Reply:
Hmm, I get where you are coming from. 

Nevertheless, I still believe stopping is the best because it reduces zombie connections to our server and thus reduces the load. 

Being a notebook means the connection can be open for a long time.

Reply:
We've removed the message to stop the object if it is initialized in a read-only state. Still going ahead and stopping it might be confusing to the user
https://neptune-labs.atlassian.net/browse/NPT-12696